tests/tests/test-xcm.ts

```diff
@@ -0,0 +1,75 @@
+import Keyring from "@polkadot/keyring";
+import { expect } from "chai";
+import { BN } from "@polkadot/util";
+
+import { ALITH_PRIV_KEY } from "../util/constants";
+import { describeDevMoonbeam } from "../util/setup-dev-tests";
+import { createBlockWithExtrinsic } from "../util/substrate-rpc";
+import { createBlock } from "typescript";
+import { customRequest } from "../tests/util";
+
+const palletId = "0x6D6f646c617373746d6E67720000000000000000";
+
+const assetMetadata = {
+  name: "DOT",
+  symbol: "DOT",
+  decimals: new BN(12),
+  isFrozen: false,
+};
+
+//TODO Gorka showed me to think of this as { parents: 1, intherior: here }. Are these the same?
+const sourceLocation = { XCM: { X1: "Parent" } };
```

Comment:
*scratching my head* In the test I wrote for the prachain test I use:
`const sourceLocationRelay = { XCM: { parents: 1, interior: "Here" } };`

@girazoki ?

Reply:
What Joshy wrote seems to me as v0, while your version is v1. Does Joshys test work? I think in that specific extrinsic we just allow v1

Reply:
My test does not work. I'll use what Antoine wrote. I copied that line from https://github.com/PureStake/moonbeam/blob/681ba3412e6198e1b47bb8cb2c2d1deb6d0cd968/tests/tests/test-asset-manager.ts#L17